deploy/helm/dynatrace-oneagent-operator/templates/Common/customresource.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: dynatrace.com/v1alpha1
+kind: OneAgent
+metadata:
+  name: oneagent
+  namespace: {{ .Release.Namespace }}
+spec:
+  apiUrl: {{ required "ApiUrl needs to be set!" .Values.oneagent.apiUrl}}
+  skipCertCheck: false
+  tolerations:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/master
+    operator: Exists
+  args:
+  - APP_LOG_CONTENT_ACCESS=1
+  env:
+  - name: ONEAGENT_ENABLE_VOLUME_STORAGE
```

Comment:
Probably leave it as an option? I know it's required for GKE and COS, but depends on the platform.

Reply:
Since this will also work with non GKE/COS platforms we set this as a default.
But you are right, we could also have this as a parameter